megablocks/layers/arguments.py

```diff
@@ -23,8 +23,12 @@ class Arguments:
     moe_lbl_in_fp32 : bool = False
 
     # Parallelism arguments.
+    #
+    # TODO(tgale): Delete the separate flag for EMP.
     moe_expert_model_parallelism : bool = False
     expert_parallel_group : Optional[torch.distributed.ProcessGroup] = None
+    moe_weight_parallelism : bool = False
```

Comment:
Isn't `moe_weight_parallelism == weight_parallel_group is not None`?
Why have both flags?

Reply:
PyTorch passes 'None' as the process group to indicate the default process group should be used. So, having weight_parallel_group == None or expert_model_parallel_group == None does not indicate that you do not want to do expert or weight parallelism. Hence the flags.